### PR TITLE
build-script: use just-built swiftc for testing purposes too.

### PR DESCRIPTION
**Pull request by nkcsgexi**

### CODE REVIEW

**Comment by nkcsgexi** [comment]
@swift-ci please test

File: build-script.py [hunk 1]
```diff
@@ -187,7 +187,7 @@ def build_swiftsyntax(swift_build_exec, swiftc_exec, build_dir, build_test_util,
 ## Testing
 
 def run_tests(swift_test_exec, build_dir, release, swift_build_exec,
-              filecheck_exec, swiftc_exec, swift_syntax_test_exec, verbose):
+              filecheck_exec, swiftc_exec, swiftc_exec_unreal, swift_syntax_test_exec, verbose):
```
Comment (jrose-apple) [inline build-script.py]: What does "unreal" mean here?

Comment (nkcsgexi) [reply build-script.py]: it's the opposite of 'realpath' in the script.

**Comment by nkcsgexi** [comment]
@swift-ci please test

**Pull request merged**